src/components/OSCALMarkdownProse.test.js

```diff
@@ -43,3 +43,23 @@ test(`OSCALMarkdownProse converts images to HTML`, () => {
   );
   expect(testerString).toEqual("<p><code>Inserting Tester Code</code></p>");
 });
+
+test(`OSCALMarkdownProse converts script to HTML`, () => {
+  const testerString = asHtml(
+    <OSCALMarkdownProse text="<script>alert('Nasty script!')</script>" />
+  );
+
+  expect(testerString).toEqual(
+    "&lt;script&gt;alert(&#x27;Nasty script!&#x27;)&lt;/script&gt;"
+  );
+});
+
+test(`OSCALMarkdownProse converts a multiline string to HTML`, () => {
+  const multilineMarkdown =
+    "this should b here # h1 Heading 8-    ## h2 Heading     This is in heading 2    ### h3 Heading      This is in heading 3    #### h4 Heading      ##### **h5 Heading**    ###### h6 Heading ";
+  const multilineAsHTML =
+    "<p>this should b here # h1 Heading 8-    ## h2 Heading     This is in heading 2    ### h3 Heading      This is in heading 3    #### h4 Heading      ##### <strong>h5 Heading</strong>    ###### h6 Heading</p>";
```

Comment:
So uhh despite being called "multiline" this is just one line.

```suggestion
  const multilineMarkdown = `
  this should be here
  # h1 heading
  
  ## h2 Heading
  
  This is in heading 2
  
  ### h3 Heading
  
  This is in heading 3
  
  #### **h5 Heading**
  `;
  
  const multilineHtml = "<p>this should be here</p><h1>h1 heading</h1><h2>h2 Heading</h2><p>This is in heading 2</p><h3>h3 Heading</h3><p>This is in heading 3 </p><h5><strong>h5 Heading</strong></h5>";
```

ya know +/- some formatting and spacing and all that